products.go

```diff
@@ -38,6 +38,8 @@ type Product struct {
 	Variants Variants
 	// Vendor is the name of the products vendor
 	Vendor string
+	// Images is a list of product image objects, each one representing an image associated with the product.
+	Images Images
```

Comment:
you seem to have combined your two PRs, did you mean to?

Reply:
yeah it is because we need the image object for the collections